binary-quantization-mistral/bq-mistral-embed.md

```diff
@@ -0,0 +1,537 @@
+
+By executing a Python command like `client.get_collection(collection_name=collection_name).vectors_count`, you are invoking a method to fetch details about a specific collection, identified by `collection_name`. Among the details returned, `vectors_count` is of particular interest as it reveals the total number of vectors stored in the collection.
+
+The output of `['']` suggests that either the code snippet did not execute as expected, or the collection currently contains zero vectors. This insight would typically prompt further investigation to ensure that data ingestion processes are functioning correctly or to validate that the specified collection is the intended target for interaction. Understanding and monitoring the vectors count is critical for both operational management and performance optimization within Qdrant, making it an essential tool in the arsenal of any data engineer or scientist working with this technology.
+
+```python
+collection_info = client.get_collection(collection_name=collection_name)
+collection_info.vectors_count
+```
+
+### Configuring Indexing
+
+In this segment, the procedure focuses on enhancing the search operation within a certain collection by adjusting its indexing settings through the `update_collection` method. Specifically, it updates the configuration of a collection (identified by `collection_name`) to adjust its `indexing_threshold`. The `indexing_threshold` parameter is crucial as it determines the number of vectors that should accumulate before the indexing process is triggered. By setting the `indexing_threshold` to 20,000, it implies that the collection will start indexing its contents once the count of stored vectors reaches 20,000. This operation is facilitated with the assistance of `models.OptimizersConfigDiff`, a part of the Qdrant client library designed to fine-tune the performance and efficiency of vector searches.
+
+The presented code doesn't produce a visible output (`['']`), suggesting that the operation completed without returning any direct data. However, implicitly, this operation modifies the configuration of the collection with respect to its indexing behavior, defining how and when the vectors stored within the collection are indexed. This step is significant as proper indexing is key to achieving faster and more efficient vector searches, particularly in large-scale datasets where search performance can be critical. By tailoring the `indexing_threshold`, users can balance between the immediacy of indexing and resource utilization, optimizing the collection's performance for specific use cases or constraints.
+
+```python
+client.update_collection(
+    collection_name=f"{collection_name}",
+    optimizer_config=models.OptimizersConfigDiff(indexing_threshold=20000),
+)
+```
```

Comment:
I doubt that we needed to disable indexing at all (100k vectors are not enough to require such measurements), why don't we just keep the defaults?

I might understand the desire to show the possibility, but not sure whether this article is the best place to do it.